ci-operator/step-registry/ibmcloud/provision/kms-key/ibmcloud-provision-kms-key-commands.sh

```diff
@@ -24,6 +24,34 @@ function create_resource_group() {
     "${IBMCLOUD_CLI}" target -g ${rg} || return 1
 }
 
+function createKey() {
+    local instance_name=$1
+    local region=$2
+    local keyInfoFile=$3
+    local id key_name key_meterial tmpFile keyCRN keyID keyInfo
+    echo -n "" > ${keyInfoFile}
```

Comment:
I think we do not need this line.

Reply:
I reuse the same ${keyInfoFile}, so clean up the previous content at the begining.

Reply:
If follow https://github.com/openshift/release/pull/47656/files#r1458196261, the keyInfoFile will be always cleaned up before writing data.